apps/mobile/src/features/send/send-form/hooks/use-send-form-btc.tsx

```diff
@@ -95,63 +94,61 @@ export function useSendFormBtc() {
     onInitSendTransfer({ utxos, feeRates }: SendFormBtcContext, values: SendFormBtcSchema) {
       try {
         const { amount: inputAmount, recipient: recipientAddress, feeRate } = values;
+        const recipient = createBitcoinAddress(recipientAddress);
+        // TODO LEA-1852 - move to form schema validation
+        // 1. validate precision
```

Comment:
Annotating the steps for now to assist with refactoring here

Reply:
Thanks so much @pete-watters, made a note to include in the refactored version.